### DISCUSSION TITLE
I deactivated my Visual Studio Enterprise subscription by mistake

### DISCUSSION THREAD

**Discussion by Habramh5** [microsoft/SATechnicalOnboarding#71] [Questions]
I deactivated my Visual Studio Enterprise subscription by mistake and I would like to reactivate it.
I followed all the instructions to reactivate this but it still doesn't work.

Please help me
![React](https://github.com/microsoft/SATechnicalOnboarding/assets/140288184/737cfbac-53e8-48b8-a9d2-aca994a15c20)

**Comment by AnthonyByansi** [comment]
Hey @Habramh5. Please submit a support ticket with your account details to the SA-Coordinator via teams and he will assist you in reactivating your Visual Studio Enterprise subscription.
